### DISCUSSION TITLE
华硕路由器ac68u访问速度慢，播放卡顿

### DISCUSSION THREAD

**Discussion by lqy93** [messense/aliyundrive-webdav#289] [Q&A]
本人华硕ac68u路由器升级版，刷梅林固件，安装ali webdav,家中网速100M。用infuse连接这个webdav后平均速度只有10-15M（2000kbps以下），播放或者拖动高码率视频卡顿，且加载也慢。
速度测试的曲线大约是一阵一阵的，高一阵子然后突然降到0，持续几秒又升上去，又降到0，循环往复。平均下来只有10-15M。
另外我的端口号设置的8080（不知道会不会影响？），其他设置均为默认。
请问这种情况正常吗？**这是因为网速不行还是路由器性能不行？或者是阿里云限速了？**其他人是否有遇到这个状况？

**Comment by messense** [comment]
对比下直接在电脑上运行的速度。

**Comment by Asongguo** [comment]
我家ac68u也是这样的问题，放到nas上的速度倒是可以

**Comment by quartets** [comment]
我家的netgear 6300 v2也是这样，两个cpu轮流到100%的利用率。所以是路由器小马拉大车了。我改用windows 命令行运行webdav，性能就很好了。现在的问题是2小时的影片，文件在60gb以上，infuse播放就卡顿。不知道是不是阿里云盘的下行速率还是有限。

**Comment by fengchen-github** [comment]
我的7621也是类似的情况，用htop就能看出来，其中一个cpu核经常100%

**Comment by Eric-SK** [comment]
ac66ub1也是一样的现象

**Comment by sungangtao** [comment]
我用的R4S软路由，infuse挂载阿里云盘，也是这个情况，测速一下0，一下80多M，换了媳妇儿的阿里账号网速一下子就上去了，感觉是不是阿里对老用户限速了

**Reply by quartets** [reply]
谢谢。这给我指明了方向。我换了一个帐号，暂时解决了播放速度问题。
